src/Avalonia.Visuals/Media/Imaging/RenderTargetBitmap.cs

```diff
@@ -21,21 +23,27 @@ public class RenderTargetBitmap : Bitmap, IDisposable, IRenderTarget
         /// <param name="dpiX">The horizontal DPI of the bitmap.</param>
         /// <param name="dpiY">The vertical DPI of the bitmap.</param>
         public RenderTargetBitmap(int pixelWidth, int pixelHeight, double dpiX = 96, double dpiY = 96)
-            : base(CreateImpl(pixelWidth, pixelHeight, dpiX, dpiY))
+           : this(RefCountable.Create(CreateImpl(pixelWidth, pixelHeight, dpiX, dpiY)))
         {
         }
 
+        private RenderTargetBitmap(IRef<IRenderTargetBitmapImpl> impl) : base(impl)
+        {
+            PlatformImpl = impl;
+        }
+
         /// <summary>
         /// Gets the platform-specific bitmap implementation.
         /// </summary>
-        public new IRenderTargetBitmapImpl PlatformImpl => (IRenderTargetBitmapImpl)base.PlatformImpl;
+        public new IRef<IRenderTargetBitmapImpl> PlatformImpl { get; }
 
         /// <summary>
         /// Disposes of the bitmap.
         /// </summary>
         public void Dispose()
```

Comment:
This `Dispose` is hiding `Bitmap.Dispose` - did you mean to mark it `override`? Should probably implement the dispose pattern in `Bitmap`.

Reply:
I think it was like that before this PR. Still need to fix it, yes.